### PR TITLE
Permit user to specify an image tag

### PR DESCRIPTION
**Pull request by funkypenguin**
This change avoids "hard-coding" the template to use the current chart's appVersion, but rather allows the user to override the tag value, and "falls back" to the AppVersion if this is undefined. This should be a non-breaking change for the current chart.

### CODE REVIEW

**Comment by gabibbo97** [comment]
After I merge #27  can you change version.yml

P.S. you are the guy from https://geek-cookbook.funkypenguin.co.nz/ ? I'm a fan!

File: charts/keycloak-gatekeeper/templates/deployment.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccountName: {{ include "keycloak-gatekeeper.serviceAccountName" . }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.repository | default .Chart.AppVersion }}"
```
Comment (timothyclarke) [inline charts/keycloak-gatekeeper/templates/deployment.yaml:28]: Shouldn't that be `image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"`   or similar ?

Comment (funkypenguin) [reply charts/keycloak-gatekeeper/templates/deployment.yaml:28]: Oops. You're right, that's a silly typo!

(And yes, I'm the geekchef ;) )

Comment (gabibbo97) [reply charts/keycloak-gatekeeper/templates/deployment.yaml]: Thanks, didn't catch that

**Pull request merged**